client/homebrew/pages/userPage/brewItem/brewItem.jsx

```diff
@@ -26,7 +26,7 @@ const BrewItem = createClass({
 			if(!confirm('Are you REALLY sure? You will lose editor access to this document.')) return;
 		}
 
-		request.get(`/api/remove/${this.props.brew.editId}`)
+		request.del(`/api/${this.props.brew.editId}`)
```

Comment:
Can we change this for clarity?
```suggestion
		request.delete(`/api/${this.props.brew.editId}`)
```